locales/id_ID/LC_MESSAGES/duckduckgo.po

```diff
@@ -582,7 +582,7 @@ msgstr "Jelajahi Seperti Biasa"
 msgid ""
 "Browse as usual while we add privacy protection. We bundled our search "
 "engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
-msgstr ""
+msgstr "Telusuri seperti biasa selagi kami menambahkan perlindungan privasi. Kami menggabungkan mesin pencarian, pemblokir pelacak, dan penegak enkripsi kami menjadi satu %s% s Ekstensi%s."
```

Comment:
@markapruett There was a typo (an extra space in a placeholder tag), that's why the CI failed. This should fix it.

```suggestion
msgstr "Telusuri seperti biasa selagi kami menambahkan perlindungan privasi. Kami menggabungkan mesin pencarian, pemblokir pelacak, dan penegak enkripsi kami menjadi satu %s%s Ekstensi%s."
```